storage/fanout.go

```diff
@@ -39,6 +40,24 @@ func NewFanout(logger log.Logger, primary Storage, secondaries ...Storage) Stora
 	}
 }
 
+func (f *fanout) StartTime() (int64, error) {
+	firstTime, err := f.primary.StartTime()
```

Comment:
Maybe add a small comment explaining what this does ?

Reply:
Done